app/gen-server/ApiServer.ts

```diff
@@ -193,10 +194,20 @@ export class ApiServer {
     // DELETE /api/orgs/:oid
     // Delete the specified org and all included workspaces and docs.
     this._app.delete('/api/orgs/:oid', expressWrap(async (req, res) => {
-      const org = getOrgKey(req);
-      const {data, ...result} = await this._dbManager.deleteOrg(getScope(req), org);
-      if (data) { this._logDeleteSiteEvents(req, data); }
-      return sendReply(req, res, {...result, data: data?.id});
+      const orgKey = getOrgKey(req);
```

Comment:
This implementation gives up on running the full deletion within a transaction, since it could take a long time, and needs to be coordinated with removals outside of the database.